pkgs/servers/monitoring/alertmanager-executor/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "alertmanager-executor";
+  version = "unstable-2022-02-04";
+
+  src = fetchFromGitHub {
+    owner = "imgix";
+    repo = "prometheus-am-executor";
+    rev = "d43ae740ade7641258cc13c4ef2b9c19e7c63fc2";
+    sha256 = "sha256-TJIh+mNBTOauheIlC4O6FI5xBzmcXgMyVA+zUFyYY/4=";
+  };
+
+  vendorSha256 = "sha256-NUu97fpCw1YPyRyHq4+sX9IiSHb3ALlx8yo4zvppP/A=";
+
+  postInstall = ''
+    mkdir -p $out/share
+    cp -r $src/examples $out/share/
+  '';
+
+  meta = with lib; {
+    description = "Execute command based on Prometheus alerts";
+    longDescription = "Also known as prometheus-am-executor.";
```

Comment:
This is not that long :^) I think we should be more verbose here.

Reply:
Fixed in [781541d](https://github.com/NixOS/nixpkgs/pull/189119/commits/781541d1c526b19c484f31b0020a9c75902f30b4)